tools/gen_objcdoc.sh

```diff
@@ -83,3 +86,8 @@ if [[ $EXPECTED_CLASSES != $ACTUAL_CLASSES ]]; then
   diff <(echo "$EXPECTED_CLASSES") <(echo "$ACTUAL_CLASSES")
   exit -1
 fi
+
+# Create the final zip file.
+pushd $OUTPUT_DIR
+zip -r "$ZIP_DESTINATION/ios-objcdoc.zip" .
```

Comment:
Will the current recipe fail if a file already exists at this path?

Reply:
No, ios-objcdoc is generated in a temp file before uploading. If the file exists the zip command overrides it.